src/esp/physics/PhysicsManager.cpp

```diff
@@ -481,24 +482,26 @@ void PhysicsManager::setObjectBBDraw(int physObjectID,
                                      DrawableGroup* drawables,
                                      bool drawBB) {
   assertIDValidity(physObjectID);
-  if (existingObjects_[physObjectID]->BBNode_ && !drawBB) {
+  if (existingObjects_.at(physObjectID)->BBNode_ && !drawBB) {
     // destroy the node
-    delete existingObjects_[physObjectID]->BBNode_;
-    existingObjects_[physObjectID]->BBNode_ = nullptr;
-  } else if (drawBB && existingObjects_[physObjectID]->visualNode_) {
+    delete existingObjects_.at(physObjectID)->BBNode_;
+    existingObjects_.at(physObjectID)->BBNode_ = nullptr;
+  } else if (drawBB && existingObjects_.at(physObjectID)->visualNode_) {
     // add a new BBNode
-    Magnum::Vector3 scale =
-        existingObjects_[physObjectID]->visualNode_->getCumulativeBB().size() /
-        2.0;
-    existingObjects_[physObjectID]->BBNode_ =
-        &existingObjects_[physObjectID]->visualNode_->createChild();
-    existingObjects_[physObjectID]->BBNode_->MagnumObject::setScaling(scale);
-    existingObjects_[physObjectID]->BBNode_->MagnumObject::setTranslation(
-        existingObjects_[physObjectID]
-            ->visualNode_->getCumulativeBB()
-            .center());
+    Magnum::Vector3 scale = existingObjects_.at(physObjectID)
+                                ->visualNode_->getCumulativeBB()
+                                .size() /
+                            2.0;
+    existingObjects_.at(physObjectID)->BBNode_ =
+        &existingObjects_.at(physObjectID)->visualNode_->createChild();
+    existingObjects_.at(physObjectID)->BBNode_->MagnumObject::setScaling(scale);
+    existingObjects_.at(physObjectID)
+        ->BBNode_->MagnumObject::setTranslation(
+            existingObjects_[physObjectID]
+                ->visualNode_->getCumulativeBB()
+                .center());
     resourceManager_->addPrimitiveToDrawables(
-        0, *existingObjects_[physObjectID]->BBNode_, drawables);
+        0, *existingObjects_.at(physObjectID)->BBNode_, drawables);
```

Comment:
The move from `[]` to `at()` is unrelated to the Attributes-related changes, maybe do it in a separate PR to have this easier to review? 

Just a suggestion, I'm not the one to enforce any rules here :)